src/masternode-payments.cpp

```diff
@@ -723,6 +723,8 @@ bool CMasternodePayments::ProcessBlock(int nBlockHeight, CConnman& connman)
     // if we have not enough data about masternodes.
     if(!masternodeSync.IsMasternodeListSynced()) return false;
 
+    CheckPreviousBlockVotes(nBlockHeight - 1);
```

Comment:
`CheckPreviousBlockVotes` does nothing for masternode payments logic (which is the job of `ProcessBlock` and that's why it's wrapped with `if(!fMasterNode)`), it just logs some additional info in debug mode. IMO it should be called from `UpdatedBlockTip` (and allow to see debug output on all nodes, not only on masternodes btw).